test/Microsoft.Dnx.Testing.CommonUtils/DnxSdk/Dnu.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Dnx.Testing
+{
+    public class Dnu
+    {
+        private readonly string _sdkPath;
```

Comment:
Are there any important ones that I should look into right now specifically? I'm not too familiar with all the options and arguments for every command so I was thinking that I will make changes whenever the need arises when porting new tests?